task/maven/0.3/maven.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: maven
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
```suggestion
    tekton.dev/pipelines.minVersion: "0.17.0"
```

Reply:
Done

Reply:
I've done the rebase. How can we proceed ?

Reply:
@jpmaida, I can still se 12 commits, can you please squash them into one single commit? 🙂

Reply:
Please check if it is right. Sorry for the mess, I am not used to this practice.

Reply:
@vinamra28